File: beginner/python/2003.py
while True:
    try:
        horario = input()
        hora, minuto = map(int, horario.split(":"))

        if 5 <= hora <= 6:
            print("Atraso maximo: 0")
        elif hora == 7:
            print("Atraso maximo:", minuto)
        elif hora == 8:
            print("Atraso maximo:", 60 + minuto)
        elif hora == 9:
            print("Atraso maximo:", 120 + minuto)
    except EOFError:
        break

File: beginner/javascript/2003.js
const readline = require('readline');
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.on('line', (input) => {
  const [hora, minuto] = input.split(':').map(Number);

  if (hora >= 5 && hora <= 6) {
    console.log("Atraso maximo: 0");
  } else if (hora === 7) {
    console.log("Atraso maximo:", minuto);
  } else if (hora === 8) {
    console.log("Atraso maximo:", 60 + minuto);
  } else if (hora === 9) {
    console.log("Atraso maximo:", 120 + minuto);
  }
});

rl.on('close', () => {
  process.exit(0);
});
